### ISSUE TITLE
Is test augmentation possible?

### ISSUE THREAD

**Issue by sacalo** [facebookresearch/maskrcnn-benchmark#596]
Is bbox and mask test time augmentation supported? I don't find any config option or function to do it, like in https://github.com/roytseng-tw/Detectron.pytorch/blob/master/lib/core/test.py#L192 or https://github.com/roytseng-tw/Detectron.pytorch/blob/master/configs/baselines/e2e_mask_rcnn_X-152-32x8d-FPN-IN5k_1.44x.yaml#L54

**Comment by fmassa** [comment]
Test augmentation is not implemented in `maskrcnn-benchmark` at the moment, but it is not hard to add a simple version on top of what is currently implemented.

**Comment by AdanMora** [comment]
HI @fmassa, I have already implemented RandomRotation (with three fill techniques, default that is black, average color or the original image as background) and ColorJitter, the rotations I had to do it manipulating the binary mask, it's the easy way I think, I included more variables to the default config file, I think we could add a new section in that file for data augmentation hyper-parameters.
